examples/local/env.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+export CLUSTER_NAME=g8s
+export COMMON_DOMAIN=${CLUSTER_NAME}.local
```

Comment:
I don't think this is always the case, `CLUSTER_NAME` and `COMMON_DOMAIN` are completely unrelated in the setup with `aws-operator` I believe.

Reply:
There is a connection between `cert-operator` and `aws-operator`. The `COMMON_DOMAIN` used should match the one in `aws-operator`. This is so the certs are valid and match the Route 53 domains. 

It also means you can configure `kubectl` to connect to the cluster. I'll update the wiki page to also include `cert-operator` as I missed this step.

Reply:
Ok thank you both, I've updated `COMMON_DOMAIN` to not depend on `CLUSTER_NAME`, let me know if anything else is needed here